.github/workflows/main.yml

```
@@ -7,23 +7,16 @@ jobs:
     runs-on: ubuntu-18.04
     name: Evaluator job
     steps:
-      - name: Wait for build to succeed
-        uses: betrybe/action-wait-for-check@v2
-        id: wait-for-build
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          checkName: codeclimate
-          ref: ${{ github.event.pull_request.head.sha || github.sha }}
       - uses: actions/checkout@v2
-        if: steps.wait-for-build.outputs.conclusion == 'success'
       - name: Cypress evaluator step
         id: cypress
-        uses: betrybe/cypress-evaluator-action@v1
-        if: steps.wait-for-build.outputs.conclusion == 'success'
+        uses: betrybe/cypress-evaluator-action@v2
+        with:
+          repository-test-name: 'betrybe/sd-0x-block6-project-facebook-signup-tests'
+          repository-test-branch: 'master'
       - name: Store evaluation step
-        uses: betrybe/store-evaluation-action@v1
-        if: steps.wait-for-build.outputs.conclusion == 'success'
+        uses: betrybe/store-evaluation-action@v2
         with:
           evaluation-data: ${{ steps.cypress.outputs.result }}
           environment: staging
-          pr-number: ${{ steps.cypress.outputs.pr-number }}
+          pr-number: ${{ steps.cypress.outputs.pr-number }}
```

```suggestion
          pr-number: ${{ github.event.number }}
```

É a mesma coisa, fica mais simples e tira uma dependência do avaliador

---

Jean é meu pastor e nada me faltará. Repete aí @renatosousafilho

---

Louvado seja o nome de Jean, o primeiro de sua casa, Senhor dos commits, Lorde do Cypress, Soberano do GitHub Actions.

---

😂 😂 😂 😂 😂